scripts/default.sh

```diff
@@ -2,6 +2,9 @@
 
 export DEBIAN_FRONTEND=noninteractive
 
+# https://github.com/hashicorp/packer/issues/41
+sleep 30
```

Comment:
:( :( :(

Not a big deal, but always so sad when this is necessary. That issue looks...rough

Reply:
Yes it's really not ideal, the same bug affects the current `master` branch.

It's definitely preferable to have a `sleep` statement than intermittently failing builds 🤷‍♂